.github/workflows/ci-python-sdk.yaml

```diff
@@ -1,21 +1,22 @@
-name: Build and test astro
+name: Build and test astro Python SDK
 on:
   push:
     branches: [ 'main', 'release-**' ]
     paths:
       - 'python-sdk/**'
       - '.github/workflows/ci-python-sdk.yaml'
-      - '*'
```

Comment:
To revert this change - since we want the tests to run if we change pre-commit hook etc